phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -629,6 +629,9 @@ public static Configuration setUpConfigForMiniCluster(Configuration conf, ReadOn
         conf.setInt(HConstants.HBASE_CLIENT_RETRIES_NUMBER, 2);
         conf.setInt(NUM_CONCURRENT_INDEX_WRITER_THREADS_CONF_KEY, 1);
         conf.setInt(GLOBAL_INDEX_ROW_AGE_THRESHOLD_TO_DELETE_MS_ATTRIB, 0);
+        if (conf.getLong(QueryServices.UNGROUPED_AGGREGATE_PAGE_SIZE_IN_ROWS, 0) == 0) {
+            conf.setLong(QueryServices.UNGROUPED_AGGREGATE_PAGE_SIZE_IN_ROWS, 2);
```

Comment:
In tests we don't generally read too many rows anyways, so won't setting this config just unnecessarily slow them down?

Reply:
No, it does not. Instead of writing a small number of new tests, by setting the page size to a very small number (e.g., 2), I make sure almost all of the existing tests exercise this new feature.